Makefile

```diff
@@ -138,7 +142,7 @@ docs:
 	sphinx-build docs/src dist/docs
 
 cpp_example: all
-	g++ ${OPTIMIZATION_LEVEL} examples/app-cpp/payload_app.cc -o examples/app-cpp/payload_app ${VENDOR_cJSON_INCLUDES} -I /usr/local/include/ -I ${CPP_LIB_DIR}/include -I ${OUTPUT_GEN_DIR} -L ${OUTPUT_LIB_DIR} -lantaris_api -lpthread ${GRPC_CPP_ADDITIONAL_LIBS} -lpython3.10;
+	g++ ${OPTIMIZATION_LEVEL} -o examples/app-cpp/payload_app  $(SRCS) ${VENDOR_cJSON_INCLUDES} -I /usr/local/include/ -I ${CPP_LIB_DIR}/include -I ${OUTPUT_GEN_DIR} -L ${OUTPUT_LIB_DIR} -lantaris_api -lpthread ${GRPC_CPP_ADDITIONAL_LIBS} -lpython3.10;
```

Comment:
@santosh-kavhar what is this operation? Will this build both payload_app directory and whatever mentioned in SRC? I think we should only have SRC. By default it will have it assigned to app-cpp. User can change it if required.

Reply:
**examples/app-cpp/payload_app** is the name and the location of binary that will be created. By this line : **SRC_DIR = examples/app-cpp**, we are telling the directory as app-cpp inside the examples directory.
So, if user wants to change it, should we have any variable inside our makefile which reads from the terminal the value set for $USER_FOLDER ?